tests/test_pipenv.py

```diff
@@ -243,6 +243,32 @@ def test_basic_dev_install(self):
             c = p.pipenv('run python -m requests.help')
             assert c.return_code == 0
 
+    @pytest.mark.dev
+    @pytest.mark.install
+    def test_install_without_dev(self):
+        """Ensure that running `pipenv install` doesn't install dev packages"""
+        with PipenvInstance() as p:
+            with open(p.pipfile_path, 'w') as f:
+                contents = """
+[packages]
+tablib = "*"
+
+[dev-packages]
+requests = "*"
+                """.strip()
+                f.write(contents)
+            c = p.pipenv('install')
+            assert c.return_code == 0
+            assert 'tablib' in p.pipfile['packages']
+            assert 'requests' in p.pipfile['dev-packages']
+            assert 'tablib' in p.lockfile['default']
+            assert 'requests' in p.lockfile['develop']
+            c = p.pipenv('run python -m requests.help')
```

Comment:
We probably want to find another way to test this. If requests changes its API around help we could mask failures here.

Parsing the response from pip freeze is ugly but an option. We could also try importing requests and asserting that fails.